client/driver/rkt_test.go

```diff
@@ -0,0 +1,102 @@
+package driver
+
+import (
+        "fmt"
+        "os"
+        "os/exec"
+        "testing"
+
+        "github.com/hashicorp/nomad/client/config"
+        "github.com/hashicorp/nomad/nomad/structs"
+
+        ctestutils "github.com/hashicorp/nomad/client/testutil"
+)
+
+// rktLocated looks to see whether rkt binaries are available on this system
+// before we try to run tests. We may need to tweak this for cross-OS support
+// but I think this should work on *nix at least.
+func rktLocated() bool {
+        _, err := exec.Command("rkt", "version").CombinedOutput()
+        return err == nil
+}
```

Comment:
Can we actually combine this with the `RocketCompatible()` function? Seems to be very closely related.